core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.ts

```diff
@@ -70,18 +70,15 @@ angular.module('oppia').component('editorNavigation', {
         SiteAnalyticsService.registerClickHelpButtonEvent(explorationId);
         var EDITOR_TUTORIAL_MODE = 'editor';
         var TRANSLATION_TUTORIAL_MODE = 'translation';
-        $uibModal.open({
-          template: require(
-            'pages/exploration-editor-page/modal-templates/' +
-            'help-modal.template.html'),
+        NgbModal.open(HelpModalComponent, {
           backdrop: true,
-          controller: 'HelpModalController',
           windowClass: 'oppia-help-modal'
         }).result.then(mode => {
           if (mode === EDITOR_TUTORIAL_MODE) {
             StateTutorialFirstTimeService.onOpenEditorTutorial.emit();
           } else if (mode === TRANSLATION_TUTORIAL_MODE) {
             StateTutorialFirstTimeService.onOpenTranslationTutorial.emit();
+            $rootScope.$applyAsync();
           }
```

Comment:
You may need to move the `applyAsync` to outside the `else if`. Go to the settings tab and try to click on the `editor tutorial`. Does it run as you expect?

Reply:
Sorry for late response, actually working on another PR's.
Done now PTAL.